source/stylesheets/refills/_cards-basic.scss

```diff
@@ -0,0 +1,45 @@
+.cards-basic {
+  @include display(flex);
+  @include flex-wrap(wrap);
+  @include justify-content(space-between);
+
+  .card {
+    @include flex-basis(20em);
+    @include flex-grow(1);
+    margin: 0 1em 0 1em;
+
+    .card-image {
+      overflow: hidden;
+      max-height: 150px;
+
+      img {
+        width: 100%;
+      }
+    }
+
+    .card-header {
+    }
+
+    .card-copy {
+
+      p {
+      }
+    }
+    
+    &:focus,
+    &:hover {
+      
+      .card-image img {
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 4